js/views/modals/About/About.js

```diff
@@ -0,0 +1,100 @@
+import $ from 'jquery';
+import app from '../../../app';
+import loadTemplate from '../../../utils/loadTemplate';
+import BaseModal from '../BaseModal';
+import Story from './Story';
+import Contributors from './Contributors';
+import Donations from './Donations';
+import License from './License';
+
+export default class extends BaseModal {
+  constructor(options = {}) {
+    const opts = {
+      removeOnClose: true,
+      removeOnRoute: false,
```

Comment:
The `removeOnRoute` option should be removed from here, so the default of 'true' will be enacted by via the baseModal. The options when set to true, ensures the modal will be removed if we route to another page.

I know you copied this code from settings, but the settings modal had some special reason to override the default.